### PR TITLE
Updated 'oc env' help message

### PR DESCRIPTION
**Pull request by pravisankar**

### CODE REVIEW

**Comment by pravisankar** [comment]
@fabianofranz PTAL

**Comment by fabianofranz** [comment]
Calling my most favorite consultant for grammar-related PR's to help review this, @liggitt.

File: pkg/cmd/cli/cmd/env.go [hunk 1]
```diff
@@ -38,18 +39,19 @@ syntax.`
   // List the environment variables defined on all pods
   $ %[1]s env pods --all --list
 
-  // Output a YAML object with updated enviroment for deployment 'registry'
+  // Output a YAML object with updated environment for deployment 'registry'
```
Comment (liggitt) [inline pkg/cmd/cli/cmd/env.go]: deployment config

File: pkg/cmd/cli/cmd/env.go [hunk 2]
```diff
@@ -38,18 +39,19 @@ syntax.`
...
   // Does not alter the object on the server
   $ %[1]s env dc/registry STORAGE_DIR=/local -o yaml
```
Comment (liggitt) [inline pkg/cmd/cli/cmd/env.go]: change the sample path name so there is no confusion that the name "local" has anything to do with it not hitting the server

File: pkg/cmd/cli/cmd/env.go [hunk 3]
```diff
@@ -38,18 +39,19 @@ syntax.`
   // List the environment variables defined on all pods
   $ %[1]s env pods --all --list
 
-  // Output a YAML object with updated enviroment for deployment 'registry'
+  // Output a YAML object with updated environment for deployment 'registry'
   // Does not alter the object on the server
```
Comment (liggitt) [inline pkg/cmd/cli/cmd/env.go]: Outputs the modified deployment config, and does not alter the object on the server

File: pkg/cmd/cli/cmd/env.go [hunk 4]
```diff
@@ -38,18 +39,19 @@ syntax.`
...
   $ %[1]s env dc/registry STORAGE_DIR=/local -o yaml
 
   // Update all replication controllers in the project to have ENV=prod
   $ %[1]s env replicationControllers --all ENV=prod
```
Comment (liggitt) [inline pkg/cmd/cli/cmd/env.go]: `rc`?

File: pkg/cmd/cli/cmd/env.go [hunk 5]
```diff
@@ -38,18 +39,19 @@ syntax.`
   // List the environment variables defined on all pods
   $ %[1]s env pods --all --list
 
-  // Output a YAML object with updated enviroment for deployment 'registry'
+  // Output a YAML object with updated environment for deployment 'registry'
   // Does not alter the object on the server
   $ %[1]s env dc/registry STORAGE_DIR=/local -o yaml
 
   // Update all replication controllers in the project to have ENV=prod
```
Comment (liggitt) [inline pkg/cmd/cli/cmd/env.go]: all containers in all replication controllers?

File: pkg/cmd/cli/cmd/env.go [hunk 6]
```diff
@@ -38,18 +39,19 @@ syntax.`
...
   $ %[1]s env replicationControllers --all ENV=prod
 
-  // Remove the enviroment variable ENV from all deployment configs
+  // Remove the environment variable ENV from all deployment configs
   $ %[1]s env deploymentConfigs --all ENV-
```
Comment (liggitt) [inline pkg/cmd/cli/cmd/env.go]: `dc`?

File: pkg/cmd/cli/cmd/env.go [hunk 7]
```diff
@@ -38,18 +39,19 @@ syntax.`
...
 
-  // Remove the enviroment variable ENV from a pod definition on disk and update the pod on the server
-  $ %[1]s env -f pod.json ENV-
+  // Remove the environment variable ENV from a deployment config definition on disk and
+  // update the deployment config on the server
+  $ %[1]s env -f dc.json ENV-
 
   // Set some of the local shell environment into a deployment on the server
```
Comment (liggitt) [inline pkg/cmd/cli/cmd/env.go]: deployment config

File: pkg/cmd/cli/cmd/env.go [hunk 8]
```diff
@@ -18,13 +18,14 @@ import (
...
 Each container in a pod may have its own environment variable definitions. This command can
-add, update, or remove environment variables from containers in pods or any object that has
-a pod template (replication controllers or deployment configurations). You can specify a
-single object or multiple, and alter environment on all containers or just those that match
-a wildcard.
+add, update, or remove environment variables from containers for any object that has
+a pod template (replication controllers or deployment configurations).
+You can list environment variables in pods or any object that has a pod template.
+You can specify a single object or multiple, and alter environment on all containers or
+just those that match a wildcard.
```
Comment (liggitt) [inline pkg/cmd/cli/cmd/env.go]: make sure at least one of the examples shows wildcarding in action

File: pkg/cmd/cli/cmd/env.go [hunk 9]
```diff
@@ -18,13 +18,14 @@ import (
 )
 
 const (
-	envLong = `Update the environment on a pod
+	envLong = `Update environment variables on a pod template
 
 Each container in a pod may have its own environment variable definitions. This command can
```
Comment (liggitt) [inline pkg/cmd/cli/cmd/env.go]: s/may have/has/

Comment (liggitt) [inline pkg/cmd/cli/cmd/env.go]: actually, maybe strike the whole first sentence, and add a little whites?
```
List environment variable definitions in one or more pods or pod templates.
Add, update, or remove container environment variable definitions in one or more
pod templates (within replication controllers or deployment configurations).
View or modify the environment variable definitions on all containers in
the specified pods or pod templates, or just those that match a wildcard.
```

**Comment by pravisankar** [comment]
[merge]

**Comment by pravisankar** [comment]
re[merge]

**Pull request merged**